torch/_inductor/pattern_matcher.py

```diff
@@ -814,10 +814,15 @@ def run_node(self, node) -> Any:
             last_node = min(indices, key=lambda tup: tup[0])[1]
 
         def percolate_tags(node, recompute_tag):
-            for arg in node.all_input_nodes:
-                if hasattr(arg, "meta"):
+            queue = [node]
```

Comment:
previously, the first `node` would not have `arg.meta["recompute"] = recompute_tag` set. was that intentional?

Reply:
Yes, that was.